src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -373,10 +465,19 @@ ApiGlobe.prototype.setRealisticLightingOn = function setRealisticLightingOn(valu
  * @params {visible} boolean.
  */
 
-ApiGlobe.prototype.setLayerVisibility = function setLayerVisibility(id, visible) {
-    this.scene.getMap().setLayerVisibility(id, visible);
-    this.update();
-    eventLayerChangedVisible.layerId = id;
+ApiGlobe.prototype.setLayerVisibility = function setLayerVisibility(layer, visible) {
```

Comment:
In this PR the documentation has been deleted or is no longer correct.

Even if apiGlobe is deleted and functions are moved, the documentation will always be associated with these functions and must therefore be up to date. 
The doc has been maintained to date and should be maintained for the benefit of all.